### DISCUSSION TITLE
Feature Add new file

### DISCUSSION THREAD

**Discussion by irvanalfi** [MohmmedAshraf/laravel-translations#107] [Ideas]
Can you add a feature to add translation files in database for add file lang folder laravel other than the files that are already available?

**Comment by angtheod** [comment]
Hello, this would be great. I thought during translations:import it would scan the lang dir and find all the files that exist in there, but it looks like it adds in the translation_files table only the 4 default laravel files.

**Reply by irvanalfi** [reply]
Yes, that's right.

**Comment by beelalamin** [comment]
waiting for this too, 

The way around this is you add the file (eg /en/home.php) and add corressponding record in the ltu_translations_files table manually . then run `php artisan translations:import --fresh`  and the file should be there in the dropdown list.
